src/core/hle/kernel/thread.h

```diff
@@ -345,8 +347,11 @@ class Thread final : public SynchronizationObject {
 
     void SetStatus(ThreadStatus new_status);
 
-    u64 GetLastRunningTicks() const {
-        return last_running_ticks;
+    constexpr s64 GetLastScheduledTick() const {
```

Comment:
```suggestion
    s64 GetLastScheduledTick() const {
```

Thread will never be constructed in a constexpr manner. Ditto for the other added functions with constexpr specifiers.